dimod/binary_quadratic_model.py

```diff
@@ -166,6 +166,7 @@ class for the K4 complete graph, where the nodes have biases
        concatenations of the node pairs (e.g., 23 for u,v = 2,3).
 
        >>> import dimod
+       ...
```

Comment:
I think it's safe to drop `import <module>` from examples in documentation of `<module>`.

Reply:
I agree